src/services/UpdateTechRecordService.ts

```diff
@@ -21,4 +21,20 @@ export class UpdateTechRecordService {
 
         return LambdaService.invoke(Configuration.getInstance().getEndpoints().functions.updateTechRecordStatus.name, event);
     }
+
+   public static updateEuVehicleCategory(vin: string, euVehicleCategory: string) {
+        const event = {
+            path: `/vehicles/${vin}/update-eu-vehicle-category`,
+            pathParameters: {
+                vin,
```

Comment:
Depending on when this gets merged, will need to be conscious of systemNumber change. Not a blocker at present.